Jenkinsfile

```diff
@@ -49,9 +49,9 @@ pipeline {
             }
         }
     }
-    post {
-        always {
-            cleanWs()
-        }
-    }
```

Comment:
Why this change? The indentation seemed correct before

Reply:
We were testing something on the CI and it potentially got messed up